### ISSUE TITLE
Add some screenshots to the README

### ISSUE THREAD

**Issue by anoadragon453** [arnav-t/riot-embedded#25]
Screenshots in the README give people an idea of whether this project will work for them.

Screenshots of the client embedded into actual websites would be cool too.

**Comment by arnav-t** [comment]
Yes, that is a good idea. That will make the README a lot more interesting to look at.

**Comment by indevi** [comment]
hello,
i am just getting started with my contributions .I found this issue ,sort-of, under my caliber .Can u please guide me through?

**Comment by nzetsu** [comment]
Hi @ShivamTyagi12345, 
If you want we could do this together. After setting everything we could test it and take screenshots and update the README. What do you think?

**Comment by indevi** [comment]
This issue totally slipped from my mind .Thanks @DiNaLo45 .Sure .Lets get started
